src/ISeabotConfig.ts

```diff
@@ -0,0 +1,14 @@
+export default interface ISeabotConfig {
+    userVoiceChannels: {
+        groupId: string;
+        triggerChannelId: string;
+    };
+    autoDeleteMessagesInChannels: Array<{ id: string; frequency: number }> | undefined;
```

Comment:
I corrected this by adding a Duration class to be used for frequencies. That way it can always be clear which unit the duration is in, and we can always get its value as milliseconds.